src/coreclr/jit/emitxarch.cpp

```diff
@@ -2671,8 +2667,7 @@ inline size_t insCodeRM(instruction ins)
 }
 
 // clang-format off
-const static
-size_t          insCodesMI[] =
+const static size_t insCodesMI[] =
```

Comment:
There are 8 instructions larger than `uint32_t` and that's `imul_08` through `imul_15`. This is because they hardcode the `REX.X` bit

We should fix that to add the REX.X bit normally.

Reply:
Same here, Fixing this would save us `~2920` more bytes